src/query/src/qSqlParser.c

```diff
@@ -632,7 +632,7 @@ tSqlExpr *tSqlExprClone(tSqlExpr *pSrc) {
   tVariantAssign(&pExpr->value, &pSrc->value);
 
   //we don't clone paramList now because clone is only used for between/and
-  assert(pSrc->Expr.paramList == NULL);
+  pSrc->Expr.paramList = NULL;
```

Comment:
change assert to assignment?

Reply:
discussed with Wei pan, setting the pExpr->Expr.paramList to NULL(not copying the paramlist pointer)